docs/data/data-grid/style/style.md

```diff
@@ -122,6 +122,7 @@ By default, the Data Grid uses the Material UI `theme.palette.background.defaul
 
 ```tsx
 import { createTheme } from '@mui/material/styles';
+import type {} from "@mui/x-data-grid/themeAugmentation";
```

Comment:
this fixes #14115, though it is already mentioned here https://mui.com/x/react-data-grid/getting-started/#typescript

Reply:
For #14115, I don't think that this PR solves it, IMHO, https://github.com/mui/mui-x/issues/14115#issuecomment-2271923988

Reply:
will do this in a separate PR 👍